src/widgets/layout/model/constants.ts

```diff
@@ -0,0 +1 @@
+export const APP_TITLE = 'TrainBrain';
```

Comment:
`constants` should be in `config` segment. And if you want just create a `config.ts` file without a dir and files inside